### ISSUE TITLE
Tab and Nav active lines need a change of color

### ISSUE THREAD

**Issue by emiliano-cicero** [liferay/clay#5374]
## What are the steps to reproduce?
This is an accessibility bug related to any tab or navigation component that uses a line as main element for showing the active state of the tab/nav. 

## What is the expected result?
Following the WCGA rules, the line used for the active state should have a contrast of at least 3:1

## Jira ticket
this issue is related to: https://issues.liferay.com/browse/LPS-168127

## Proposal
My proposal is to add a new color (Primary-L0) between the 2 that we currently have.
To get this new color I applied 14% of Lighten to the $Primary color and has a minimum color contrast of 3.15:1.
<img width="491" alt="image" src="https://user-images.githubusercontent.com/27729173/218806175-91efae52-4789-494e-b206-d4070743317a.png">
